mixin-utils/test/Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: tests
+
+jsonnetfile.lock.json: jsonnetfile.json
+	jb install
+
+vendor: jsonnetfile.lock.json
+	jb install
```

This'll just execute the same command twice, right?

---

Nope, `make` seems smart enough, at least my version GNU Make 4.2.1

---

oh wait, you mean if the lock changed.
In my test it does run it again, but jb install is fast, because it doesn't download stuff

---

I'd simply check for `vendor/`, `jb install` will create `jsonnetfile.lock.json` either way.

---

I am inspired by the use of Make in this way, might use it elsewhere. ;)

---

I think this'll prevent `jb install` being executed twice:

```suggestion
vendor jsonnetfile.lock.json: jsonnetfile.json
	jb install
```

---

nice, even shorter, thanks!